plutus-tx-plugin/test/Budget/andCheap.budget.golden

```diff
@@ -0,0 +1,2 @@
+({cpu: 10994100
```

Comment:
Am I reading this right and `andCheap` is more expensive than `andExpensive`? lol

Reply:
I guess the `True` and `False` cases don't have the exact same cost, but not sure.